src/main/java/pages/ArticleCreateEditPage.java

```diff
@@ -0,0 +1,87 @@
+package pages;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class contains the webelements of the login form
+ */
+public class ArticleCreateEditPage extends BasePage {
+
+    @FindBy(className = "editor-page")
+    private WebElement panelCreateEditorArticle;
+
+    @FindBy(xpath = "//input[@placeholder='Article Title']")
+    private WebElement inputArticleTitle;
+
+    @FindBy(xpath = "//input[@placeholder=\"What's this article about?\"]")
+    private WebElement inputArticleAbout;
+
+    @FindBy(xpath = "//textarea[@placeholder='Write your article (in markdown)']")
+    private WebElement inputArticleContent;
+
+    @FindBy(xpath = "//input[@placeholder='Enter tags']")
+    private WebElement inputArticleTags;
+
+    @FindBy(xpath = "//button[normalize-space()='Publish Article']")
+    private WebElement buttonPublishArticle;
+
+    @FindBy(xpath = "//*[@class='error-messages']/li")
+    protected List<WebElement> listErrorMsg;
+
+    public ArticleCreateEditPage() {
+        super();
+    }
+
+    public boolean getPanelFormArticle() {
+        return panelCreateEditorArticle.isDisplayed();
+    }
+
+    public void writeArticleTitle(String title) {
+        sendKeystoElement(inputArticleTitle, title);
+    }
+
+    public void clearFlieds() {
+        int con = 10;
+        do {
+            inputArticleTitle.clear();
+            inputArticleAbout.clear();
+            inputArticleContent.sendKeys(Keys.chord(Keys.CONTROL,"a", Keys.DELETE));
+            inputArticleTags.clear();
+            forceWait();
+            con--;
+        } while (!(inputArticleTitle.getAttribute(ATTRIBUTE_VALUE).isEmpty()
+                && inputArticleAbout.getAttribute(ATTRIBUTE_VALUE).isEmpty()
+                && inputArticleContent.getText().isEmpty()
+                && inputArticleTags.getAttribute(ATTRIBUTE_VALUE).isEmpty())||con==0);
+    }
```

Comment:
We talked about this implementation today. I don't think it's friendly enough 🤔 What do you think? How can we improve this?

Reply:
I can create a method of cleaning problem fields to have maintainable code, I would improve it in the following card